src/main/java/appeng/parts/reporting/AbstractPartMonitor.java

```diff
@@ -79,6 +79,7 @@
 {
 	private static final IWideReadableNumberConverter NUMBER_CONVERTER = ReadableNumberConverter.INSTANCE;
 	private IAEItemStack configuredItem;
+	private String itemStackSize;
```

Comment:
I think the naming of the variable is quite bad since it is not the item stack size being stored (which would natively be an integer) but a render cache or something along the line. Maybe just `textRenderCache` or so.